main.js

```diff
@@ -0,0 +1,20 @@
+const todo = ['掃除', '買い物', '散歩'];
+
+const add = () => {
+  console.log ('====================');
```

Comment:
こちらも細かい点なのですが、()の前にスペースが空いており、このスペースは不要です。
```suggestion
  console.log('====================');
```

このような構文のミスを自動で補完してくれるツールがあり、我々から[prettier](https://prettier.io/)というものをご紹介させていただいています。
設定方法の記事も載せておきますので、参考になさってください。
https://zenn.dev/k_kazukiiiiii/articles/670ebae0005872

もしご不明な点があれば、遠慮なくご質問ください！